$OpenBSD: patch-src_joystick_bsd_SDL_sysjoystick_c,v 1.11 2012/05/16 14:01:19 edd Exp $

It is easy to exhaust uhid devices in this day and age.

--- src/joystick/bsd/SDL_sysjoystick.c.orig	Tue May 15 17:09:10 2012
+++ src/joystick/bsd/SDL_sysjoystick.c	Tue May 15 17:09:29 2012
@@ -77,7 +77,7 @@
 #include "../SDL_sysjoystick.h"
 #include "../SDL_joystick_c.h"
 
-#define MAX_UHID_JOYS	4
+#define MAX_UHID_JOYS	16
 #define MAX_JOY_JOYS	2
 #define MAX_JOYS	(MAX_UHID_JOYS + MAX_JOY_JOYS)
 
